src/OrbitGl/TimerTrack.cpp

```diff
@@ -146,55 +144,60 @@ void TimerTrack::DrawTimer(TextBox* prev_text_box, TextBox* current_text_box,
   double elapsed_us = end_us - start_us;
 
   // Draw the Timer's text if it is not collapsed.
-  if (!is_collapsed) {
+  if (!draw_data.is_collapsed) {
     // For overlaps, let us make the textbox just a bit wider:
     double left_overlap_width_us = start_or_prev_end_us - start_us;
     double text_x_start_us = start_or_prev_end_us - (.25 * left_overlap_width_us);
     double right_overlap_width_us = end_us - end_or_next_start_us;
     double text_x_end_us = end_or_next_start_us + (.25 * right_overlap_width_us);
 
-    bool is_visible_width =
-        (text_x_end_us - text_x_start_us) * inv_time_window * canvas->GetWidth() > 1;
-    WorldXInfo world_x_info =
-        ToWorldX(text_x_start_us, text_x_end_us, inv_time_window, world_start_x, world_width);
+    bool is_visible_width = (text_x_end_us - text_x_start_us) * draw_data.inv_time_window *
+                                draw_data.canvas->GetWidth() >
+                            1;
```

Comment:
Nit: Stupid clang-format thing, could I ask you to put the 1 in the previous line

Reply:
Done. Needed parentheses around the calculation.